composer.json

```diff
@@ -9,7 +9,7 @@
     "laravel/tinker": "^1.0",
     "aws/aws-sdk-php-laravel": "^3.1",
     "doctrine/dbal": "~2.5.13",
-    "dosomething/gateway": "^1.1.6",
+    "dosomething/gateway": "^1.1.8",
```

Comment:
The caret should take care of this, but let's set this to `^1.14.8` to be a bit clearer!